_posts/2023-10-11-october-bitdevs.md

```diff
@@ -0,0 +1,37 @@
+---
+layout: post
+type: socratic
+title: "BitDevs Amsterdam Conference Edition, October 11th 2023"
+meetup: https://www.meetup.com/bitdevs-amsterdam/
+published: true
+---
+
+## BitDevs Amsterdam meeting 007!
+
+It's time for a special edition of BitDevs, _The BitDevs Conference Edition!_ We hope to have a lot of friends from out of town joining, so we'll be meeting at a different place and different time than usual! Please read the details below carefully!
+
+### Wednesday, October 11th 2023 @ 3PM
```

Comment:
Let's make this @2PM.

Reply:
I think this should still be the event start time, not the door open time. This is consistent with how we've done it previously.